marlowe-dashboard-server/src/Marlowe/Run/Wallet/V1/Types.hs

```diff
@@ -0,0 +1,50 @@
+{-# LANGUAGE DeriveAnyClass     #-}
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE DerivingStrategies #-}
+module Marlowe.Run.Wallet.V1.Types where
+
+import Cardano.Prelude
+
+import qualified Cardano.Wallet.Primitive.Types as Cardano
+import qualified Data.Aeson as Aeson
+import Data.Aeson.Types (parseFail)
+import qualified Data.Text as T
+import Data.Text.Class (FromText (..), TextDecodingError (..), ToText (..))
+import Ledger.Address (PaymentPubKeyHash)
+import Servant (FromHttpApiData (..))
+
+newtype WalletId = WalletId { unWalletId :: Cardano.WalletId }
```

Comment:
Why do we define WalletId here? By doing this aren't we making a difference between the WalletId needed by the PAB and the one used by the dashboard-server?

Reply:
For the instances (the wrapped type does not provide them)